gradescopeCronJob/gradescope_to_spreadsheet.py

```diff
@@ -4,9 +4,7 @@
 
 from fullGSapi.api import client as GradescopeClient
 import os.path
-import sys
```

Comment:
Put this back. It is used in the code

Reply:
We had a duplicate `import sys` statement. I removed one of them

Reply:
Sorry I might be blind, what line is the `import sys` statement in now?